tests/tools/utils.py

```diff
@@ -118,6 +118,9 @@ def _chunk_graph(
     metadata["graph_name"] = name
     metadata["node_type"] = g.ntypes
 
+    # add node_type_counts
+    metadata["node_type_counts"] = [g.num_nodes(ntype) for ntype in g.ntypes]
```

Comment:
the name looks not intuitive to me. seems indicates the count of node types instead of the num of nodes for each node type. could you use a more clarified name?

Reply:
How about "num_nodes_per_type" ??